Source/WebCore/platform/glib/SettingsStateGLib.h

```diff
@@ -25,16 +26,12 @@
 
 #pragma once
 
+#include <optional>
 #include <wtf/text/WTFString.h>
 
-namespace IPC {
-class Decoder;
-class Encoder;
-}
+namespace WebCore {
 
-namespace WebKit {
-
-struct GtkSettingsState {
+struct SettingsStateGLib {
```

Comment:
SettingsState is confusing, since we have Settings class, I would use SystemSettingsState. I think this could be moved to SystemSettings.h to make it even clearer that this belongs to SystemSettings.